blockchain/validate.go

```diff
@@ -803,6 +823,40 @@ func (b *BlockChain) checkBlockContext(block *btcutil.Block, prevNode *blockNode
 				return err
 			}
 		}
+
+		// If the CSV soft-fork isn't fully active, then we can skip
+		// the sequence locks checks below.
+		if csvState != ThresholdActive {
+			return nil
+		}
+
+		// Load all of the utxos referenced by the inputs for all
+		// transactions in the block don't already exist in the utxo
+		// view from the database. This view is required in order to
+		// properly evalulate the sequence locks (relative lock-time)
+		// for each input referenced within the block.
+		utxoView := NewUtxoViewpoint()
```

Comment:
After looking at this again, this section isn't even needed.  The code later on in `checkConnectBlock` (with my other suggested corrections), will enable the script flag `txscript.ScriptVerifyCheckSequenceVerify` which instructs the script engine to enforce the locks.